src/schema.js

```diff
@@ -202,16 +214,31 @@ export default class DataStructureGenerator {
         element.description = new StringElement(schema.description);
       }
 
-      if (schema.default !== undefined && !_.isArray(schema.default) &&
-          !_.isObject(schema.default)) {
+      let def = schema.default;
```

Comment:
Can we call this `default` not `def` to remove the abbreviation? Clearer during reading. As a Pythonist `def` is shorthand for `define` too so doesn't feel right.

Reply:
`default` is reserved. Can't use it.